javascript/src/test/components/ChannelMessages/MessageScrollable.test.js

```diff
@@ -1,116 +0,0 @@
-import React from "react";
-import { render } from "@testing-library/react";
-import MessageScrollableView from "main/components/ChannelMessages/MessageScrollableView";
-import useSWR from "swr";
-import MessageListView from "../../../main/components/ChannelMessages/MessageListView";
```

Comment:
Same thing for this file. The file is incorrectly named (it should be `MessageScrollableView.test.js`), but we probably shouldn't be removing others' tests.